From 7405cefc7198d839168cda8ec58c6b337c6d5e09 Mon Sep 17 00:00:00 2001
From: Todd <toddrme2178@gmail.com>
Date: Thu, 12 Dec 2019 13:16:36 -0500
Subject: [PATCH] Fix cmake file installation directory

It was hard-coded as being installed in a platform-independent directory rather than using the variables the user defined.
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f3150cde..621a39fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,15 +70,15 @@ write_basic_package_version_file(
 configure_package_config_file(
     cmake/${PROJECT_NAME}Config.cmake.in
     cmake/${PROJECT_NAME}Config.cmake
-    INSTALL_DESTINATION lib/cmake/)
+    INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/)
 
 #   Installation
 
-install(EXPORT ${PROJECT_NAME}Targets DESTINATION lib/cmake)
+install(EXPORT ${PROJECT_NAME}Targets DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
 install(FILES
         ${CMAKE_BINARY_DIR}/blitz.pc
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(FILES
         ${CMAKE_CURRENT_BINARY_DIR}/cmake/${PROJECT_NAME}Config.cmake
         ${CMAKE_CURRENT_BINARY_DIR}/cmake/${PROJECT_NAME}ConfigVersion.cmake
-        DESTINATION lib/cmake/)
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/)
